dluhc-component-library/src/components/timetableForm/components/DescriptionPage.tsx

```diff
@@ -1,13 +1,18 @@
 import { Textarea } from "src/components/formComponents";
 
-const DescriptionPage = () => {
+interface DescriptionPageProps {
+  value: string | undefined;
+  onChange: (value: string) => void;
+}
+
+const DescriptionPage = ({ value, onChange }: DescriptionPageProps) => {
   return (
     <div className="flex flex-col">
       <h1 className="my-6 text-3xl font-bold">Summary text</h1>
       <p className="text-sm mb-2 text-gray-600">
         Keep this specific to your own Local Plan
       </p>
-      <Textarea value="" onChange={() => {}} maxLength={400} />
+      <Textarea value={value || ""} onChange={onChange} maxLength={400} />
```

Comment:
You are having to do a lot of `||`s and ` | undefined` to catch the initial state, its probably worth having an initialState that just has the values in